pkg/plugins/runtime/k8s/metadata/annotations_test.go

```diff
@@ -55,6 +55,53 @@ var _ = Describe("Kubernetes Annotations", func() {
 		})
 	})
 
+	Describe("GetBoolean()", func() {
+		type testCase struct {
+			input    string
+			expected bool
+		}
+		annotations := map[string]string{
+			"key1": "yes",
+			"key2": "no",
+			"key3": "true",
+			"key4": "false",
```

Comment:
probably need an error cases like:

```suggestion
			"key4": "false",
			"key5": "foo",
```

Reply:
I'll add, thanks!